org.lflang/src/org/lflang/generator/GeneratorBase.xtend

```diff
@@ -1564,29 +1565,37 @@ abstract class GeneratorBase extends AbstractLFValidator implements TargetTypes
             if (mainReactor.host !== null) {
```

Comment:
Does this if statement interfere with the intended object of the added code? It seems like if no `at` clause is present on the main/federated reactor, the following code will not execute.

Reply:
ah, you're right. good catch.